critiquebrainz/frontend/views/bb_literary_work.py

```diff
@@ -0,0 +1,62 @@
+from flask import Blueprint, render_template, request
+from flask_login import current_user
+from werkzeug.exceptions import NotFound, BadRequest
+from flask_babel import gettext
+import critiquebrainz.db.review as db_review
+import critiquebrainz.frontend.external.bookbrainz_db.literary_work as bb_literary_work
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+from critiquebrainz.frontend.views import get_avg_rating, LITERARY_WORK_REVIEWS_LIMIT
+
+bb_literary_work_bp = Blueprint('bb_literary_work', __name__)
+
+
+@bb_literary_work_bp.route('/<uuid:id>')
+def entity(id):
+    id = str(id)
+    literary_work = bb_literary_work.get_literary_work_by_bbid(id)
+
+    if literary_work is None:
+        raise NotFound(gettext("Sorry, we couldn't find a literary work with that BookBrainz ID."))
```

Comment:
while we use 'literary work' as our identifier for this type, perhaps here we should just say "Sorry, we couldn't find a BookBrainz work with this ID"